routes.js

```diff
@@ -68,7 +68,7 @@ function fastifySwagger (fastify, opts, next) {
     schema: { hide: true },
     handler: function (req, reply) {
       const file = req.params['*']
-      if (file === '') {
+      if (file === 'null') {
```

Comment:
why this change? I think this was correct before

Reply:
That's why I asked you to take a look. Actually, otherwise, I can't reach the code inside the `if` statement. I believe that's because it uses the default redirection rules defined above.

How do you suggest to change this line so it a) works correctly, b) does not harm the coverage?

Reply:
I think you just need to send your http request to `/documentation` or `/documentation` so that the `*` param would be empty.

Reply:
Nope, it should be smth like: `/documentation/a` at least in order to trigger those lines. Therefore, they won't be empty. 
That's why I've changed that check to 
```
if (file === 'null')
```

I tried different approaches but still couldn't cover those lines 'as is'. So, if you have any ideas just let me know

Reply:
What is called when you call `/documentation` or `/documentation/` then?

Reply:
Code on line `23` if I understand correctly:
```
fastify.route({
    url: '/',
    method: 'GET',
    schema: { hide: true },
    handler: (req, reply) => {
      reply.redirect(getRedirectPathForTheRootRoute(req.raw.url))
    }
  })
```

Reply:
Then we can drop the if.